contracts/entry-point/src/execute.rs

```diff
@@ -551,12 +564,17 @@ fn verify_and_create_fee_swap_msg(
     ibc_fee_coin: &Coin,
 ) -> ContractResult<WasmMsg> {
     // Validate swap operations
-    validate_swap_operations(
-        &fee_swap.operations,
-        remaining_asset.denom(),
-        &ibc_fee_coin.denom,
-    )?;
+    if fee_swap.routes.is_empty() {
```

Comment:
I think this check is redundant with the len() != 1 check below, right? (meaning len != 1 check will catch this already) If so down to just keep the len != 1 check and remove this is empty check